src/redirects.js

```diff
@@ -33,7 +33,7 @@ const createDashboardRedirects = (environment) => {
   ];
 };
 
-const getRedirects = (environment) => [
+const getRedirects = (environment, language) => [
```

Comment:
Would it be an idea to add a default value?

Reply:
+ no types here? 😃

Reply:
Haha I'l typescriptify it for you

And good idea